mw-webapp/src/component/waysBlock/WaysBlock.tsx

```diff
@@ -11,8 +11,8 @@ export const WaysBlock = () => {
     WayService.onValueFromRealTimeDb(setWays);
     () => {
       //TODO
-      // removeEventListener from db if needed (read about handling event listeners
-      // in react use effect components (when and whyu you shoud remove them))
+      // RemoveEventListener from db if needed (read about handling event listeners
```

Comment:
looks like this lines mean you have a memory leak, it is a bug, should you create a ticket for it?

Reply:
Create an issue #64

Reply:
Create an issue #64